test/testdata/googleapis_test/gapic/packaging/api_defaults.yaml

```diff
@@ -19,6 +19,9 @@ author: Google Inc
 email: googleapis-packages@google.com
 homepage: https://github.com/googleapis/googleapis
 license: Apache-2.0
+development_status:
```

Comment:
Ah, I see -- I think you're right. @lukesneeringer, please hold off on review of these PRs while I switch them over to use the existing configuration.